sdk/core/azure-core/src/main/java/com/azure/core/util/paging/PageRetriever.java

```diff
@@ -6,22 +6,21 @@
 import reactor.core.publisher.Flux;
 
 /**
- * A type representing the contract to retrieve one or more pages.
+ * This class handles retrieving pages from a service.
```

Comment:
Same here, the page can be retrieved from any source.

Reply:
Removed service reference